input/connect/connect_ps3.c

```diff
@@ -136,12 +136,8 @@ static void hidpad_ps3_get_buttons(void *data, input_bits_t *state)
 	struct hidpad_ps3_data *device = (struct hidpad_ps3_data*)data;
 	if ( device )
 	{
-		/*copy first 16 bits - standard RetroPad controls*/
-		BITS_COPY16_PTR(state, device->buttons);
-
-		/*PS button?*/
```

Comment:
Is this not needed for the PS Quick Menu toggle button?

Reply:
With BITS_COPY16_PTR and if (device->buttons & 0x10000) it doesn't toggle the menu and the PS button doesn't respond in input menu either.

With BITS_COPY32_PTR and the [autoconfig file](https://github.com/libretro/retroarch-joypad-autoconfig/blob/master/hid/Sony_PLAYSTATION(R)3_Controller.cfg) the PS can toggle menu by default.

In short, all ds3 buttons are working now but there may be a better way for the PS button fix.